solvers/ISTA.py

```diff
@@ -12,7 +12,7 @@ class Solver(BaseSolver):
     stopping_strategy = 'callback'
 
     # any parameter defined here is accessible as a class attribute
-    parameters = {'alpha': [1.9],
+    parameters = {'alpha': [1., 1.9],
                   'use_acceleration': [False, True]}
 
```

Comment:
After looking back at the theory, it seems that we should skip if `alpha > 1 and use_acceleration`.